agent/src/main/java/io/jenkins/plugins/remotingkafka/Agent.java

```diff
@@ -56,12 +62,45 @@ public static void main(String... args) throws InterruptedException, IOException
             LOGGER.info("Please provide a secret");
             System.exit(-1);
         }
+        if (options.kafkaUsername == null) {
```

Comment:
It would be great to continue supporting non-authenticated mode if possible.
E.g. by "-noauth" flag or by empty username.